massa-bootstrap/src/server.rs

```diff
@@ -63,14 +65,15 @@ use crate::{
 /// Abstraction layer over data produced by the listener, and transported
 /// over to the worker via a channel
 type BsConn = (Duplex, SocketAddr);
+
 /// handle on the bootstrap server
 pub struct BootstrapManager {
-    update_handle: std::thread::JoinHandle<Result<(), Box<BootstrapError>>>,
```

Comment:
While in the process of moving over to native threads and de-asyncing the code-base, I would like to be absolutely explicit about which threads are being used. Might be good to remove the std::thread import, actually.

Reply:
I can put them back but my ide displays warnings

Reply:
Given the value of the suggested change assumes it doesn't affect the ergonomics for others (aside from the fact that it's a bit ugly in the meantime) it's better to move things towards being nice in the general case, rather than making things nice for my specific use-case.

Happy with the changes you made, given your explanation. Thanks!